elip-0201.mediawiki

```diff
@@ -0,0 +1,36 @@
+<pre>
+  ELIP: 201
+  Layer: Wallets
+  Title: Dust limit reduction for Confidential Transactions ("DustCT")
+  Author: Kilian Rausch <k@bol.tz>
+  Comments-Summary: No comments yet.
+  Comments-URI: TBD
+  Status: Draft
+  Type: Standards Track
+  Created: 2025-03-07
+  License: BSD-3-Clause
+</pre>
+
+==Introduction==
+
+===Abstract===
+
+This document proposes to reduce the wallet-side enforced dust limit for Confidential Transactions on the Liquid Network.
+
+===Copyright===
+
+This document is licensed under the 3-clause BSD license.
+
+===Motivation===
+
+[https://github.com/ElementsProject/elements Elements], [https://github.com/Blockstream/gdk GDK] and [https://github.com/Blockstream/lwk LWK] ("Liquid reference wallets") currently enforce a wallet-side 546 sats dust limit, originally inherited from Bitcoin Core<ref>[https://github.com/bitcoin/bitcoin/blob/ee68b05f3d639b28b9d23b48324bc5f20ccfe49c/src/policy/policy.cpp#L36 Bitcoin Core Policy]</ref>. With ELIP 200<ref>[https://github.com/ElementsProject/ELIPs/blob/main/elip-0200.mediawiki ELIP 200]</ref> live on Liquid mainnet, network fees for Confidential Transactions dropped by ~10x, with <20 sats total for a single input, single output Confidential Transaction. The default dust limit of Liquid's reference wallet libraries should be lowered to reflect this and allow for micropayment use cases on Liquid.
```

Comment:
Not accurate for Elements. From the code in your policy link below, the dust limit is dependent on the output type and the node's `dustrelayfee`